package/gargoyle/files/www/js/common.js

```diff
@@ -12,6 +12,33 @@ var TSort_Classes = new Array ('odd', 'even'); // table sorting zebra row suppor
 var menuState = []; //nav Menu State variable
 var navTimer; //timeout for restoring menu
 
+// Abbreviation.
+function byId(id)
```

Comment:
Would like this more if we added an optional second parameter to set the document variable to something else (as is done in some other parts of Gargoyle).
e.g. function setChildText in common.js

Reply:
Done.